### ISSUE TITLE
Default consistency level not applied when manager initialized with withDefault{Write|Read|Serial}Consistency()

### ISSUE THREAD

**Issue by mykapp** [doanduyhai/Achilles#292]
Hi @doanduyhai 
I am facing a problem with the way to default consistency level. I am using Achilles version 5.2.0.

When I initialize the ManagerFactory as below : 

`ManagerFactory managerFactory = ManagerFactoryBuilder.builder(cluster)
.withDefaultKeyspaceName("myks")
.withDefaultReadConsistency(ConsistencyLevel.LOCAL_QUORUM)
.withDefaultWriteConsistency(ConsistencyLevel.LOCAL_QUORUM)
.build();`

Every query that I run are executed with consistency level = LOCAL_ONE (I can see it in the Achilles logs).				
				
If I initialize the managerFactory level with a consistencyMap as below :

`Map consistencyMap = new HashMap<String, ConsistencyLevel>();
consistencyMap.put("testing_table", ConsistencyLevel.LOCAL_QUORUM);`
`ManagerFactory managerFactory = ManagerFactoryBuilder.builder(cluster)
.withDefaultKeyspaceName("myks")
.withDefaultWriteConsistencyMap(consistencyMap)
.withDefaultReadConsistencyMap(consistencyMap)
.build();`

then the requests are correctly executed with the expected consistency level (in my case LOCAL_QUORUM).
Did I miss something in the configuration? Or is it a bug in Achilles as it does not apply correctly the default consistency levels with the methods withDefault{Write|Read|Serial}Consistency?

Regards,

**Comment by doanduyhai** [comment]
Hmm let me check the source code

**Comment by doanduyhai** [comment]
Did you read this doc ? https://github.com/doanduyhai/Achilles/wiki/Consistency-Level#settings-priority

According to the doc,

Default consistency level defined on the `com.datastax.driver.core.Cluster` object will override the default consistency level defined in Achilles configuration (`withDefaultReadConsistency(ConsistencyLevel.LOCAL_QUORUM) .withDefaultWriteConsistency(ConsistencyLevel.LOCAL_QUORUM)`)

Please check and be **sure** that you don't configure your `cluster` object with QueryOption CL = LOCAL_ONE ...

**Comment by mykapp** [comment]
I didn't read it. I will check this!

**Comment by mykapp** [comment]
Hi,
I read your doc about consistency priority and I double checked my code. 

I confirm I don't configure my cluster with any QueryOption.
My cluster is initialized with this simple line : 
```
Cluster cassandraCluster = Cluster.builder().addContactPoints(cassandraConfiguration.getHosts()).build();
```

But when I look into the variable of the instancied cassandraCluster I can see that, even if I don't set any QueryOption, the cluster object contains a default one with the following values : 
```
queryOptions = {QueryOptions@6954} 
  - consistency = {ConsistencyLevel@6958} "LOCAL_ONE"
  - serialConsistency = {ConsistencyLevel@6959} "SERIAL"
  - ....
```

According to the consistency priority of Achilles this is normal that I have a consistency level of "LOCAL_ONE" in my request.
But as the Cluster always have a default QueryOption, I think the withDefault{Write|Read|Serial}Consistency() methods of Achilles are useless.

NB : I am using Cassandra driver 3.1.3.

**Comment by doanduyhai** [comment]
> But as the Cluster always have a default QueryOption, I think the withDefault{Write|Read|Serial}Consistency() methods of Achilles are useless

This is the real issue. So maybe we need to change the consistency level priority order

**Comment by mykapp** [comment]
Thanks for all the fixes you did in 5.2.1. Everything is OK for me now.
